lib/safira_web/controllers/redeemable_controller.ex

```diff
@@ -2,16 +2,32 @@ defmodule SafiraWeb.RedeemableController do
   use SafiraWeb, :controller
 
   alias Safira.Store
+  alias Safira.Accounts
 
   action_fallback(SafiraWeb.FallbackController)
 
   def index(conn, _params) do
-    redeemables = Store.list_redeemables()
+    redeemables = Store.list_store_redeemables()
     render(conn, "index.json", redeemables: redeemables)
   end
```

Comment:
In my opinion we should list all redeemables. The logic that shows them or not based on their stock should be in frontend.